eclipse-emfcloud/modelserver-glsp-integration
# #47 Update project-template to latest versions

- Update to latest GLSP and emfcloud versions
  - Adapt to API breaks due to latest GLSP version
- Update test workspace
  - Remove workspace settings (do not force disable of auto-save in workspace)
  - Restructure example model files
- Introduce tree label provider for .tasklist files
- Update VSCode workspace (include server folders separately to access launch configs and tasks)
- Update build
  - Update outdated Jenkinsfile
  - Add checkstyle to m2 build and project-template builds

Resolves #47

---

The Jenkins job to check with newest versions currently builds this branch to fix the prior build failures due to the API breaks:
https://ci.eclipse.org/emfcloud/job/check-modelserver-glsp-integration/
